packages/common/src/dotlottie-player.ts

```diff
@@ -1726,6 +1740,27 @@ export class DotLottiePlayer {
       loop,
     }));
 
+    if (this._animation && lottieContainsAudio(this._animation)) {
+      const howlerModule = await import('howler');
+
+      if (this._howlerInstance) {
+        this._howlerInstance.unload();
+      }
+
+      // Initialize Howler instance outside of the callback so that we have instant access to it
+      this._howlerInstance = howlerModule.default.Howler;
+
+      const howl = (assetPath: string): AudioFactory => {
+        const howlInstance = new howlerModule.default.Howl({
+          src: [assetPath],
+        });
+
+        return howlInstance;
+      };
+
+      options['audioFactory'] = howl;
```

Comment:
`options` is a reference type. Appending extra properties to the user's arguments might lead to unexpected problems. This would be acceptable if it were a value type.

Could we introduce a separate variable for the `audioFactory` inside this function?